package.json

```diff
@@ -23,6 +23,7 @@
     "uint64be": "^1.0.1"
   },
   "devDependencies": {
+    "he-chat": "^1.0.0",
```

Comment:
This dependency is in a weird direction from my perspective. I would think that the extensions would depend on hypercore, rather than the other way around.